### ISSUE TITLE
Dockerfile with 2 High and 2 Low Kics vulnerabilities + 1 Medium False Positive

### ISSUE THREAD

**Issue by miguelfreitas93** [Checkmarx/kics#1680]
### Expected Behavior
0 kics vulnerabilities for Dockerfile in the root of kics repository

### Actual Behavior
2 High - True positive:
Missing USER Instruction, Severity: HIGH, Results: 2
 - Dockerfile:22
 - Dockerfile:1

1 Medium - False Positive since this query is related with AWS not Dockerfile's:
Access Key Is Not Rotated Within 90 Days, Severity: MEDIUM, Results: 1
 - Dockerfile:1

2 Low - True Positive:
Healthcheck Instruction Missing, Severity: LOW, Results: 2
 - Dockerfile:1
 - Dockerfile:22

### Steps to Reproduce the Problem

  1. git clone https://github.com/Checkmarx/kics
  2. Add docker-compose.yml file:
```yaml
version: '3'

services:
  kics:
    platform: linux
    image: checkmarx/kics:latest
    container_name: kics
    hostname: kics
    volumes:
      - ./:/path/
    build:
      context: ./
      dockerfile: Dockerfile
    command: -p /path/Dockerfile
```

  3. docker-compose up
  4. ` docker-compose up
Recreating kics ... done
Attaching to kics
kics    | Files scanned: 1
kics    | Parsed files: 1
kics    | Queries loaded: 728
kics    | Queries failed to execute: 0
kics    | Access Key Is Not Rotated Within 90 Days, Severity: MEDIUM, Results: 1
kics    |       Dockerfile:1
kics    | Healthcheck Instruction Missing, Severity: LOW, Results: 2
kics    |       Dockerfile:1
kics    |       Dockerfile:22
kics    | Missing USER Instruction, Severity: HIGH, Results: 2
kics    |       Dockerfile:22
kics    |       Dockerfile:1
kics exited with code 0`

### Specifications

  - Version: latest
  - Platform: Linux
  - Subsystem: Docker

**Comment by miguelfreitas93** [comment]
@ruigomescx I've scanned this repo: https://github.com/github/docs and I've found **698 FPs** for query **Access Key Is Not Rotated Within 90 Days, Severity: MEDIUM**

**Comment by joaoReigota1** [comment]
This issue has been fixed with PR - #1741

**Comment by miguelfreitas93** [comment]
@ruigomescx @joaoReigota1 True Positive results should be fixed as well.
Dockerfile should be refactored in order to fix these vulnerabilities:
kics        | Healthcheck Instruction Missing, Severity: LOW, Results: 2
kics        |   Dockerfile:1
kics        |   Dockerfile:22
kics        | Missing USER Instruction, Severity: HIGH, Results: 2
kics        |   Dockerfile:1
kics        |   Dockerfile:22
